mlrun/model_monitoring/helpers.py

```diff
@@ -322,3 +323,37 @@ def get_invocations_metric(project: str) -> ModelEndpointMonitoringMetric:
         name=mm_constants.PredictionsQueryConstants.INVOCATIONS,
         full_name=get_invocations_fqn(project),
     )
+
+
+def enrich_model_endpoint_with_model_uri(
+    model_endpoint: ModelEndpoint,
+    model_obj: mlrun.artifacts.ModelArtifact,
+):
+    """
+    Enrich the model endpoint object with the model uri from the model object. Instead of using the model uri that
```

Comment:
this documentation is more suitable to the jira (where we describe the fix we are planning to do).
Here we should just describe what the function do.

Reply:
@assaf758 , I just copy paste the function to different file.. but no problem I will change it :)